lib/standard/helpers/autocorrectable.rb

```diff
@@ -0,0 +1,18 @@
+module Standard
+  module Helpers
+    module Autocorrectable
+      def autocorrectable_offense?(offense)
+        cop_instance(offense.cop_name).support_autocorrect?
```

Comment:
IIRC, `standardrb --fix` only runs what-rubocop-deems-to-be "_safe_" autocorrections. Could you verify that's the case?

If so, this check should probably also only return true for safe autocorrections

Reply:
> IIRC, `standardrb --fix` only runs what-rubocop-deems-to-be "_safe_" autocorrections. Could you verify that's the case?

Yes, I believe that's exactly what [`MergesSettings`](https://github.com/testdouble/standard/blob/master/lib/standard/merges_settings.rb#L31) does

> If so, this check should probably also only return true for safe autocorrections

I partially reimplemented the [`RuboCop::Cop::AutocorrectLogic#autocorrect_enabled?`](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/autocorrect_logic.rb#L19) method in f793685362529d1e41d6ccf84aadbbd3b61c8dd7. We could use the _actual_ implementation if our default value (i.e. when the `--fix` option is not supplied) for the `safe_auto_correct` flag is `true`, but I'm not sure if that's the case (I struggled to understand from where `LoadsYamlConfig` gets its default values). 

@searls , could you help me figure this out?